install_config/cluster_metrics.adoc

```diff
@@ -112,11 +112,11 @@ API
 ----
 
 . Before it can deploy components, the *metrics-deployer* service account must
-also be granted the `edit` permission for the *openshift-infra* project:
+also be granted the `edit` permission for the *openshift-metrics* project:
```

Comment:
How does giving edit permission to a service account for a specific namespace give it root access to the whole cluster? Or is this just because of how the permissions are configured for the `openshift-infra` project?